2010-06-13  Aurelien Jarno  <aurelien@aurel32.net>

	* locale/C-translit.h.in: Add entry for U20A1.

---
 locale/C-translit.h.in              |    1 +
 localedata/locales/translit_neutral |    2 ++
 2 files changed, 3 insertions(+)

--- a/locale/C-translit.h.in
+++ b/locale/C-translit.h.in
@@ -103,6 +103,7 @@
 "\x2061"	""	/* <U2061> FUNCTION APPLICATION */
 "\x2062"	""	/* <U2062> INVISIBLE TIMES */
 "\x2063"	""	/* <U2063> INVISIBLE SEPARATOR */
+"\x20a1"	"CRC"	/* <U20A1> COLON SIGN */
 "\x20a8"	"Rs"	/* <U20A8> RUPEE SIGN */
 "\x20ac"	"EUR"	/* <U20AC> EURO SIGN */
 "\x2100"	"a/c"	/* <U2100> ACCOUNT OF */
--- a/localedata/locales/translit_neutral
+++ b/localedata/locales/translit_neutral
@@ -128,6 +128,8 @@
 <U2063> ""
 % DONG SIGN
 <U20AB> "<U0110><U1ED3><U006E><U0067>"
+% COLON SIGN
+<U20A1> "<U0043><U0052><U0043>"
 % EURO SIGN
 <U20AC> "<U0045><U0055><U0052>"
 % TRADE MARK SIGN
